test/JDBC/input/storedProcedures/Test-sp_rename-vu-prepare.sql

```diff
@@ -0,0 +1,58 @@
+-- tsql
```

Comment:
Please put this file in all upgrade schedules.

We also need dependency tests, you may name the new files as `Test-sp_rename-dep-vu-*`. The dependency prepare script should have a procedure, a function and a view that depend on sp_rename. Note that the dependency test needs to be added to latest schedule.
See https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/610/files#diff-272b1045c86970661de855a946fe1f1eb1b40db189b48e61239cc2805189855e as an example